### ISSUE TITLE
Error after update HA 10.3. It's not possible to configure loxone binary_sensor by adding platform: loxone to the binary_sensor configuration.

### ISSUE THREAD

**Issue by Koenh333** [JoDehli/PyLoxone#205]
Miniserver Gen1 
Loxone software 13.1.11.17

**Describe the bug**
I installed the Update HA to version: 
  Home Assistant 2023.10.3 
  Supervisor 2023.10.0 
  Operating System 10.5 
  Frontend-versie: 20231005.0 - latest 

**********
I get the next error afther the installation of the HA update:

It's not possible to configure loxone binary_sensor by adding platform: loxone to the binary_sensor configuration. 
Please check the documentation for more information on how to set up this integration.

To resolve this:

    Remove platform: loxone occurences from the binary_sensor: configuration in your YAML configuration file.
    Restart Home Assistant.

Example that should be removed:

binary_sensor:
  - platform: loxone

```

Please can you help me on this one? 
Every status of Loxone with 'binairy sensor' doesn't work anymore. The other entities are still good. 

Thx !


![HA_error](https://github.com/JoDehli/PyLoxone/assets/113801909/15d4b043-781e-4286-a240-424ac9222a1f)

**Comment by JoDehli** [comment]
@Koenh333 did you use the latest pyloxone version

**Comment by Koenh333** [comment]
Where can i see the installed version? 
I don't see any updates by Integrations/HACs so i think i have the last version...

**Comment by JoDehli** [comment]
@Koenh333 under hacs - pyloxone there should be the installed version listed. 

Do you have custom binary sensors in your setup?

**Comment by Koenh333** [comment]
Yes but i don't see anything from Loxone under hacs... 
It doesn't appear in the list. 


![Hacs_PyLoxone](https://github.com/JoDehli/PyLoxone/assets/113801909/0c8bd947-5a1a-4d46-8922-8514c329fa84)

**Comment by JoDehli** [comment]
@Koenh333 I asked for custom binary sensors in your yaml configuration not for custom integrations.

**Comment by Koenh333** [comment]
What exactly do you mean by 'custom'?
These were read directly from Loxone, I have not changed their name or anything. I use them directly in my dashboard.

Analog or digital signals no longer work.
Every entitiy that starts with
"light."
"Sensor."
"binary_sensor"
and "cover."
gives an error.

**Comment by JoDehli** [comment]
If you have not installed it via hacs how did you installed pyloxon3?

**Comment by Koenh333** [comment]
I installed it via hacs, I just don't understand why I don't see it anywhere under hacs...

this will probably be the problem...

I have already removed the integration and added it again, and then I find the integration and just have to apply the settings. So I also suspect that he did not completely remove Pyloxone.

**Comment by Koenh333** [comment]
I just noticed that not all binary_sensors give an error. 
Here you see a good one and one that can no longer be loaded... I don't see any difference actually...

![HA_Binary_Sensor](https://github.com/JoDehli/PyLoxone/assets/113801909/3f4d1b8d-13f1-4ca8-bcba-250a5ca4e374)

**Comment by JoDehli** [comment]
These are duplicates I think. So uninstall pyloxone via hacs. Make sure that the folder loxone is removed in the custom components folder. If there is any file or folder from the pyloxone install remove it. If you do not know how to get the folder read the doc from hacs or search on the net. 

Then restart ha without pyloxne.

Then remove all entities which where created from loxone. Restart again without pyloxone installed. 

Then install pyloxone and configure it.

**Comment by Koenh333** [comment]
Hi @JoDehli , it finally worked! 

After I deleted everything from pyloxone and restarted, it still didn't work. 
I could no longer see the pyloxone anywhere in HACS.

So I completely removed HACS itself. (integration + folders)

Then I reinstalled HACS and then I found pyloxone again. Then everything was installed again and everything works as before. Only have to check a few more entities, the rest will be in order again.

Thanks for your time and help! Have a nice Sunday!
